src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -5708,6 +5730,48 @@ switch_status_t config_sofia(sofia_config_t reload, char *profile_name)
 						} else {
 							sofia_clear_pflag(profile, PFLAG_SECURE);
 						}
+					} else if (!strcasecmp(var, "bridge-accept-cng")) {
+						if (switch_true(val)) {
+							sofia_set_pflag(profile, PFLAG_BRIDGE_ACCEPT_CNG);
+						} else {
+							sofia_clear_pflag(profile, PFLAG_BRIDGE_ACCEPT_CNG);
+						}
+					} else if (!strcasecmp(var, "partner-bridge-accept-cng")) {
+						if (switch_true(val)) {
+							sofia_set_pflag(profile, PFLAG_PARTNER_BRIDGE_ACCEPT_CNG);
+						} else {
+							sofia_clear_pflag(profile, PFLAG_PARTNER_BRIDGE_ACCEPT_CNG);
+						}
+					} else if (!strcasecmp(var, "bridge-forward-cng-interval") && !zstr(val)) {
+						profile->bridge_forward_cng_interval = switch_core_strdup(profile->pool, val);
+						sofia_set_pflag(profile, PFLAG_BRIDGE_FORWARD_CNG_INTERVAL);
```

Comment:
I don't see this flag being test in the code

Reply:
@dev-ryanc, after suggesting that we should be only using partner flags in sofia_outgoing_channel, there is no use to this flag on its own anymore. Do we want to use it elsewhere?

Reply:
What I'm asking is I don't see it being tested. Not sure if this is something that is missing or should be removed.

Reply:
We're testing this flag in sofia_glue_attach_private; we don't really use it elsewhere. That's why I've been wondering if we need half of these flags